factions/mummies/units/mummy/mummy.xml

```diff
@@ -7,7 +7,7 @@
 		<max-hp value="500" regeneration="10"/>
 		<max-ep value="0"/>
 		<armor value="10"/>
-		<armor-type value="organic"/>
+		<armor-type value="mummy"/>

```

Comment:
```suggestion
		<armor-type value="wood"/>
```

I don't see any big advantage to adding an extra armor type and making several changes to the mummy_invasion_testing.xml file. If the concern is how vulnerable the mummies are to axe-throwers, a simple armor change should be sufficient. Or change the game set-up.